src/components/contact/sections/ContactUs.jsx

```diff
@@ -0,0 +1,110 @@
+import React, { useState } from "react";
+import { Container, Col, Form, Button, Nav } from "react-bootstrap";
+
+const ContactUs = () => {
+  const initContactFormState = {
+    name: "",
+    surName: "",
```

Comment:
surName should just be surname as it is one word in english

however i would prefer line 6 be firstName and line 7 be lastName (more common than name/surname)